install/upgrade/versions/1.4.8.sh

```diff
@@ -5,3 +5,4 @@
 #######################################################################################
 #######                      Place additional commands below.                   #######
 #######################################################################################
+apt-get install libicu?? -y
```

Comment:
Running apt install / apt-get install is not possible during install of hestiacp package it will crash the server we should set it as dependency in 

https://github.com/hestiacp/hestiacp/blob/c192eaa9024cac455ebd08bf0cb6ff3a375203b2/src/deb/php/control#L9

Reply:
The depends on script accepts the double `??` as a wildcard? If yes, I can change it

Reply:
I don't know we have to try it...